cli/Cargo.toml

```diff
@@ -60,3 +60,4 @@ teeracle = []
 sidechain = []
 offchain-worker = []
 production = []
+dcap = []
```

Comment:
Why is this relevant for the cli?

Reply:
With this the `worker_features` environment variable is not required anymore, because the `additional_features` can supply it to everywhere (notably to the `enclave-runtime` besides the `worker`) and for it to work I had to add it to the `cli` as well.

Reply:
Ouch, this will never be obvious to anybody. Please document it. :)

Reply:
Added a note in 8034e585. :+1: